### DISCUSSION TITLE
Mesh model

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12594] [Q&A]
Dear contributors,
In FDS MESH command, are there any constraints for mesh size, e.g. the number of y and z directions must follow $2^l3^m5^n$ law? I am confused about this because it was mentioned in another document, but I didn't find any information in the FDS User's Guide.

Kind regards,
LI

**Comment by mcgratta** [comment]
There are no limits, however, you get slightly better performance if the number of cells in y and z are multiples of 2, 3 or 5. The reason is that the pressure solver uses fast Fourier transforms, which work best with those values.

**Reply by lihy213** [reply]
THANKS! The detailed introduction was found in the previous Users' Guide and now I understand it.
